### PR TITLE
feat: upgrade to angular 9

### LINKED ISSUE #107: Angular 9 & Ivy support
**Describe the bug**
The lib doesn't support build with Ivy engine. Then, `ngcc` can't transform it. The following error appears : 

```
Error: Error on worker #2: TypeError: Cannot read property 'injectorImports' of undefined
    at NgModuleDecoratorHandler.compile (C:\myPathTo\node_modules\@angular\compiler-cli\src\ngtsc\annotations\src\n
g_module.js:347:174)
```

**To Reproduce**
In project use `material-community-components`, execute: ` ngcc --properties es2015 browser module main --first-only --create-ivy-entry-points --target .\material-community-components\
`

**Expected behavior**
Be compatible with angular 9+ and Ivy

**Desktop (please complete the following information):**
 - OS: Window10

### PR DESCRIPTION
**Pull request by damingerdai**
## Purpose 

- Closes #107 

## Changes

- build: update angular version to 9.
- build: update angular material version to 9
- fix: ScrollingModule instead of ScrollDispatchModule

### Checklist

* **Are tests written for added code?**

- [ ] Yes
- [x] No because: no added code

* **Did you introduce any new dependencies? 

- [x] No
- [ ] Yes
  - Dependency:
  - Reason:

### CODE REVIEW

**Comment by damingerdai** [comment]
@Splaktar could you review my code? My project is using this package and i want to try to update angular 9. thanks.

**Comment by demiro** [comment]
Please make it happen guys ;)

**Comment by tiaguinho** [comment]
Hi guys!

I'm a little sick today, but tomorrow I'll review and make a new release to Angular 9 and ivy.
Thanks for understanding

**Comment by damingerdai** [comment]
> Hi guys!
> 
> I'm a little sick today, but tomorrow I'll review and make a new release to Angular 9 and ivy.
> Thanks for understanding

 i am sorry to hear that you are not well recently.

File: src/lib/tsconfig.spec.json [hunk 1]
```diff
@@ -3,7 +3,6 @@
   "compilerOptions": {
     "outDir": "./out-tsc/spec",
     "baseUrl": "./",
```
Comment (Splaktar) [inline src/lib/tsconfig.spec.json]:
```suggestion
```

**Comment by progmancod** [comment]
I really need this one too

File: src/lib/tsconfig.spec.json (hunk 1 above)
Comment (damingerdai) [reply src/lib/tsconfig.spec.json]: thanks.

**Comment by demiro** [comment]
sorry for being pushy... but any progress?

**Comment by damingerdai** [comment]
> sorry for being pushy... but any progress?

 Reviewing

**Pull request merged**